frontend/src/app/store/reducers/case-thunks.ts

```diff
@@ -98,14 +98,16 @@ const addAssessment =
         caseCode: "HWCR",
         assessmentDetails: {
           actionNotRequired: assessment.action_required === "No",
-          actions: assessment.assessment_type.map((item) => {
-            return {
-              date: assessment.date,
-              actor: assessment.officer?.value,
-              activeIndicator: true,
-              actionCode: item.value,
-            };
-          }),
+          actions: assessment.assessment_type
```

Comment:
I don't think the ternary statement here isn't necessary. If `assessment_type` is an empty array `map()` will return an empty array